package.json

```diff
@@ -135,25 +135,27 @@
     "concurrently": "^3.1.0",
     "constitute": "^1.6.2",
     "css-loader": "^0.23.1",
+    "currency-symbol-map": "^3.1.0",
     "ejs": "^2.4.1",
     "email-templates": "^2.3.0",
     "express": "^4.14.0",
     "express-force-ssl": "^0.3.1",
     "extract-text-webpack-plugin": "^0.9.1",
     "file-loader": "^0.9.0",
-    "five-bells-condition": "^3.2.0",
-    "five-bells-ledger": "^19.4.1",
-    "five-bells-shared": "^21.2.0",
+    "five-bells-condition": "^5.0.1",
+    "five-bells-ledger": "^20.1.0",
+    "five-bells-shared": "^23.0.0",
     "font-awesome": "^4.7.0",
     "font-awesome-webpack": "0.0.4",
     "hoist-non-react-statics": "^1.0.3",
     "http-proxy": "^1.12.0",
-    "ilp": "^8.0.1",
-    "ilp-connector": "^13.1.4",
+    "ilp": "git://github.com/interledgerjs/ilp#mj-currency_scale",
+    "ilp-connector": "git://github.com/interledgerjs/ilp-connector#mj-currency_scale-with-fbme",
     "ilp-kit-cli": "^11.1.0",
-    "ilp-plugin-bells": "^10.2.3",
+    "ilp-plugin-bells": "git://github.com/interledgerjs/ilp-plugin-bells#mj-currency_scale",
```

Comment:
TODO: use master versions, since the `mj-currency_scale` branch is now merged everywhere.